### ISSUE TITLE
Request: NUT 2.8.2

### ISSUE THREAD

**Issue by electroflame** [instantlinux/docker-tools#149]
[NUT 2.8.2](https://github.com/networkupstools/nut/releases/tag/v2.8.2) was released on April 1, 2024.

2.8.2 has some updates to the USB drivers (including support for a Liebert UPS I've got), among other fixes, so it'd be great if the Docker image could be updated.

**Comment by instantlinux** [comment]
The build failed security scan (outdated core packages, as listed in the [build log](https://gitlab.com/instantlinux/docker-tools/-/jobs/6697960421), so at least one more update needs to be made. Stay tuned.

**Comment by instantlinux** [comment]
There are about two dozen vulnerable packages in Alpine 3.19 base image, and I cannot figure out how to force the build to include only the new ones from edge. Instead I get two copies of each package:
```
# apk info lsblk
lsblk-2.39.3-r0 description:
Block device list tool from util-linux

lsblk-2.39.3-r0 webpage:
https://git.kernel.org/cgit/utils/util-linux/util-linux.git

lsblk-2.39.3-r0 installed size:
112 KiB

lsblk-2.40-r2 description:
Block device list tool from util-linux

lsblk-2.40-r2 webpage:
https://git.kernel.org/cgit/utils/util-linux/util-linux.git

lsblk-2.40-r2 installed size:
128 KiB
```
The security scanner complains about vulnerable lsblk version `2.39.3-r0` (and 25 other similar issues) and I don't know how to clobber those bits from the image.

Version `3.19` of the base image came out 7-Dec-2023; version `3.20` is not scheduled yet; historically these updates come out every 5 or 6 months. I hope the next one is expedited because this makes most builds vulnerable.

**Comment by instantlinux** [comment]
This update is complete, in two stages:

* 7-May: PR #153 disabled vulnerability scan, image under 3.19 published with new nut version 2.8.2
* 1-Jul: PR #160 update to 3.20.
